libcxx/cmake/Modules/HandleLibC.cmake

```diff
@@ -0,0 +1,34 @@
+#===============================================================================
+# Define targets for linking against the selected C library
+#
+# After including this file, the following targets are defined:
+# - libcxx-libc-headers: An interface target that allows getting access to the
+#                        headers of the selected C library.
+# - libcxx-libc-shared: A target representing the selected shared C library.
+# - libcxx-libm-shared: A target representing the selected shared C math library.
```

Comment:
Is it not possible to roll this up into the libc target? From the libc++ side, we'd like to have the illusion that `libm` is nothing but an implementation detail of `libc`, kind of like `-latomic`.

Reply:
Done.